stacks/workflows/deploy-all.yaml

```diff
@@ -0,0 +1,6 @@
+workflows:
```

Comment:
not sure the all in one workflow is usable yet, but that's the idea here (one-click install)